CMakeLists.txt

```diff
@@ -285,6 +285,12 @@ else()
   set(INSTALL_OS_HEADERS_DIR "include/os" CACHE PATH "Path where the headers will be installed")
 endif()
 
+if(NOT APPLE)
```

Comment:
This should not happen on Windows or Android, please explicitly check for Linux and BSD.

Reply:
You are right. We use pkgconfig on windows aswell, but this a specific case. I will change it to UNIX AND NOT APPLE

Reply:
Who is "we" in this case?  We (the swift project) definitely doesn't use pkg-config on Windows, nor does "we" (The Browser Company).